### DISCUSSION TITLE
Custom Window Pointer in Windows (I don't know a correct name for this)

### DISCUSSION THREAD

**Discussion by PieroCastillo** [AvaloniaUI/Avalonia#6586] [Q&A] [answered]
Hello, I found a simple way to [render Graphics in Windows Desktop Background](https://www.youtube.com/watch?v=MN8XbFVmGCs), and now I want to use it for Avalonia Win32, but I don't know a way to put a custom window pointer (and I don't know if that is possible), so, what u think? I want to do this for a Wallpaper App

**Comment by kekekeks** [comment, marked as answer]
I don't think that any UI toolkit allows wrapping externally managed window handles. But you can try implementing your own `ITopLevelImpl` and use `EmbeddableControlRoot` for hosting your content.
